src/material/menu/menu-trigger.ts

```diff
@@ -691,6 +686,4 @@ export abstract class _MatMenuTriggerBase implements AfterContentInit, OnDestroy
   },
   exportAs: 'matMenuTrigger',
 })
-export class MatMenuTrigger extends _MatMenuTriggerBase {
-  protected override _panelClass = 'mat-mdc-menu-panel-wrapper';
```

Comment:
why did the panel class get removed?

Reply:
I put this class in the previous PR to make this one easier to land https://github.com/angular/components/pull/27378. Now we don't need it anymore.